solution/backend/serverless-experimental.yml

```diff
@@ -48,7 +48,7 @@ provider:
     DEPLOY_NUMBER: ${env:RUN_ID}
     EUA_FEATUREFLAG: ${ssm:/eregulations/eua/featureflag}
     # Used for when you want to deploy your own
```

Comment:
is this comment needed anymore?

Reply:
Yes its good to have a comment for what that specifici arn is for.